app/src/main/java/com/example/indoor_volleyball/Activities/MainActivity.java

```diff
@@ -1,68 +1,54 @@
 package com.example.indoor_volleyball.Activities;
 
-import androidx.annotation.NonNull;
-import androidx.appcompat.app.AppCompatActivity;
-import androidx.core.app.NotificationCompat;
-import androidx.core.app.NotificationManagerCompat;
-import androidx.fragment.app.Fragment;
-import androidx.fragment.app.FragmentManager;
-import androidx.lifecycle.Observer;
-import androidx.work.Data;
-import androidx.work.ExistingPeriodicWorkPolicy;
-import androidx.work.OneTimeWorkRequest;
-import androidx.work.PeriodicWorkRequest;
-import androidx.work.WorkInfo;
-import androidx.work.WorkManager;
-import androidx.work.WorkRequest;
-
+import android.annotation.SuppressLint;
```

Comment:
Right clock on the top level folder and click optimize imports.  That'll do it for the whole project so that you don't have imports you're not using anymore!